tests/testCoadds.py

```diff
@@ -144,11 +144,11 @@ def tearDown(self):
         del self.butler
 
     def getCalexpIds(self, butler):
-        if len(butler.repository._parents) is not 1:
+        if len(butler.inputs) is not 1:
             raise RuntimeError("This test assumes one input repository, can not continue with %s"
-                               % butler.repository._parents)
+                               % butler.repository.inputs)
 
-        return butler.repository._parents[0]._mapper.index['calexp']['visit'][None]
+        return butler.inputs[0].repo._mapper.index['calexp']['visit'][None]
```

Comment:
It would be nicer if this didn't have to know about the mapper internals (or that there's even a mapper at all) and could just use a standard Butler interface.  But it isn't a big deal, as this is just a test.